CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(SKIP_GOOGLETEST_INSTALL "Skip installation" ${SKIP_GOOGLETEST_INSTALL} )
```

Comment:
Er... I think you just want to set the initial value to `OFF`... `option` won't change it if it has been set already. Although this should still work, it works more by accident (empty variables are considered as Boolean-false) than proper design.